docs/FAQ/FAQ-Apps.md

```diff
@@ -3,53 +3,101 @@
 This document covers frequently asked questions about the Apps in BTCPay.
 
 * [What are the Apps in BTCPay?](#what-are-the-apps-in-btcpay)
-* [Is there a limit on the number of Apps I can deploy?](#is-there-a-limit-on-the-number-of-apps-i-can-deploy)
+* [Is there a limit on the number of Apps I can create?](#is-there-a-limit-on-the-number-of-apps-i-can-create)
 * [Is there a Point of Sale feature in BTCPay?](#is-there-a-point-of-sale-feature-in-btcpay)
 * [How can I use BTCPay in a physical store?](#how-can-i-use-btcpay-in-a-physical-store)
 * [How to customize the appearance of Point of Sale App in BTCPay?](#how-to-customize-the-appearance-of-point-of-sale-app-in-btcpay)
 * [What is a Payment Button?](#what-is-a-payment-button)
+* [How to create a Pay Button with a custom amount?](#how-to-create-a-pay-button-with-a-custom-amount)
+* [How to map a domain name to an app?](#how-to-map-a-domain-name-to-an-app)
 * [How to fix empty spacing around payment button](#how-to-fix-empty-spacing-around-payment-button)
 * [How to integrate WooCommerce Store in BTCPay Crowdfund app?](#how-to-integrate-woocommerce-store-into-a-btcpay-crowdfund-app)
-* [How to create Pay button with a custom amount?](#how-to-create-pay-button-with-a-custom-amount)
-* [How to map a domain name to an app?](#how-to-map-a-domain-name-to-an-app)
 
 ## What are the Apps in BTCPay?
 
 Apps are plugins (features) you can use to expand the use case of your BTCPay.
 
-## Is there a limit on the number of Apps I can deploy?
+## Is there a limit on the number of Apps I can create?
 
-There's no limit. Each app can be created an unlimited amount of times. Apps are added on a store level; you need to have a store to create an app.
+There's no limit. Apps are added on the store level. You need to have a store already created, to create an app which belongs to that store. You can create as many apps as you like for each store. 
 
 ## Is there a Point of Sale feature in BTCPay?
 
 Yes. Please read our [guide on creating the POS app](../WhatsNext.md#creating-the-pay-button).
 
 ## How can I use BTCPay in a physical store?
 
-You can use our Point of Sale app. For having a physical PoS, right now, the easiest solution is to utilize PoS App that’s available and then set that as URL within your in-store POS. When you create POS app within BTCPay Server - you will get publicly accessible URL where checkout buttons for products you’ve defined will be displayed. Click on the button creates an invoice.
+You can use our Point of Sale (PoS) app. When you create a PoS app within BTCPay Server it will be publicly accessible via URL where checkout buttons for products you’ve created for your PoS will be displayed. For having a physical PoS, the easiest solution (currently) is to create a PoS App in BTCPay and display it on any web device such as a phone, tablet or pc.
```

Comment:
```suggestion
You can use our Point of Sale (PoS) app. When you create a PoS app within BTCPay Server, it will be publicly accessible via a URL where checkout buttons for products you’ve created for your PoS will be displayed.
To have a physical PoS, the easiest solution (currently) is to create a PoS App in BTCPay and display it on any web device such as a phone, tablet or pc.
```